modules/codemod/package.json

```diff
@@ -4,8 +4,7 @@
   "license": "Apache-2.0",
   "version": "0.0.0",
   "description": "A collection of codemods for use on Workday Canvas Kit packages.",
-  "main": "dist/commonjs/index.js",
-  "module": "dist/es6/index.js",
+  "main": "index.js",
```

Comment:
I removed the build step for the codemods since jscodeshift supports Typescript files and this simplifies development and packaging.